webapp/components/test-runs-query.html

```diff
@@ -205,10 +211,17 @@
           this.setProperties(batchUpdate);
         }
 
-        computeResultsRangeMessage(from, maxCount) {
+        computeResultsRangeMessage(from, to, maxCount) {
           if (from) {
-            var diff = Math.floor((Date.now() - from) / 86400000);
-            return `Showing test runs from the last ${diff} days`;
+            const max = to || Date.now();
+            var diff = Math.floor((max - from) / 86400000);
+            const shortDate = date => date.toLocaleDateString(navigator.language, {
+              month: 'short',
+              day: 'numeric',
+            });
+            return to
+              ? `Showing test runs from ${shortDate(from)} to ${shortDate(to)}`
```

Comment:
Including the year if different or not the current year here would be good. I got "Showing test runs from Dec 15 to Nov 21" where the start was 2016 and end 2018. In that case I'd expect the year to be shown for both, so for simplicity maybe always show year for both dates or neither.

Reply:
Done (when they differ)